docs/ca/dev/preparation-for-going-live.md

```diff
@@ -63,7 +63,7 @@ After pointing the domain name to your Spryker project, some of your customers m
   - Connect Staging S3 bucket to a staging environment. 
 - Implement the following performance tips:
   - Implement approaches described in all the [performance guidelines](/docs/scos/dev/guidelines/performance-guidelines/performance-guidelines.html).
-  - Implement the recommended Jenkins [performance and stability improvements](/docs/dg/dev/backend-development/cronjobs/optimizing-jenkins-execution.html) where applicable.
+  - Complete the [Jenekins Stability Checklist](/docs/ca/dev/best-practices/jenkins-stability-checklist.html)
```

Comment:
Do you want to link it to [Best practices: Jenkins stability](https://docs.spryker.com/docs/ca/dev/best-practices/best-practises-jenkins-stability.html) ?